### PR TITLE
maint: remove teleportr

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Removes teleportr since it's being deprecated.

### CODE REVIEW

File: packages/contracts-periphery/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "test": "yarn test:contracts",
     "test:contracts": "hardhat test --show-stack-traces",
     "test:forge": "forge test",
-    "test:coverage": "NODE_OPTIONS=--max_old_space_size=8192 hardhat coverage && istanbul check-coverage --statements 90 --branches 84 --functions 88 --lines 90",
+    "test:coverage": "NODE_OPTIONS=--max_old_space_size=8192 hardhat coverage && istanbul check-coverage --statements 90 --branches 82 --functions 88 --lines 90",
```
Comment (smartcontracts) [inline packages/contracts-periphery/package.json:22]: Dropped branch requirement from 84 to 82, didn't feel like this was the right place to add more tests to increase coverage

**Comment by tynes** [comment]
It looks like there is more CI config that needs to be removed

```
ci/circleci: teleportr-tests Expected — Waiting for status to be reported
```

**Review by roninjin10 — APPROVED** [review]

**Comment by roninjin10** [comment]
lgtm once you get ci to pass

File: packages/contracts-periphery/package.json (hunk 1 above)
Comment (roninjin10) [reply packages/contracts-periphery/package.json:22]: There are stories of giant corp companies not deleting code simply to stay above some arbitrary code cove numbers

**Review by tynes — APPROVED** [review]

**Pull request merged**